py/strato/common/log/configurelogging.py

```diff
@@ -72,6 +72,11 @@ def addFileHandler(name, path):
     logging.info("Logging started for %s" % name)
 
 
+def changeHandlerLogLevelbyHandlerType(logger, logLevel, handlerType=None):
+    """ if not specifying handlers type , change log logleve to all """
```

Comment:
Why it cant be null? I meant that if it is null change level for all
handlers

On Mon, Dec 28, 2015 at 7:32 PM, rom-stratoscale <notifications@github.com>
wrote:

> In py/strato/common/log/configurelogging.py
> <https://github.com/Stratoscale/pycommonlog/pull/4#discussion_r48493217>:
>
> > @@ -72,6 +72,11 @@ def addFileHandler(name, path):
> >      logging.info("Logging started for %s" % name)
> >
> >
> > +def changeHandlerLogLevelbyHandlerType(logger, logLevel, handlerType=None):
> > +    """ if not specifying handlers type , change log logleve to all """
>
> why do u set handlerType to None? It's not acceptable value.
> Just add: assert handlerType is not None and dont run the loop.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Stratoscale/pycommonlog/pull/4/files#r48493217>.
>